src/pages/city-fire-hydrant/city-fire-hydrant.html

```diff
@@ -0,0 +1,23 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>Liste des bornes fontaine</ion-title>
+  </ion-navbar>
+</ion-header>
+<ion-content>
+  <ion-list>
+    <ion-list-header>
+      Bornes
+    </ion-list-header>
+    <ion-item *ngFor="let hydrant of cityfireHydrants">
```

Comment:
Ici je pense qu'il serait bien de mettre les bornes en ordre de numéro de borne.  

J'ai déjà créé un pipe "orderBy" pour pouvoir ordonner les listes qui s'utilise comme ceci : 

`let hydrant of cityfireHydrants | orderBy:'number'`

Il suffit d'importer "PipesModule" dans le module de la page (city-fire-hydrant-module.ts j'imagine).